server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStatsResponse.java

```diff
@@ -41,30 +50,85 @@ public RemoteStoreStatsResponse(
         List<DefaultShardOperationFailedException> shardFailures
     ) {
         super(totalShards, successfulShards, failedShards, shardFailures);
-        this.shards = shards;
+        this.remoteStoreStats = shards;
```

Comment:
nit - lets rename the function argument shard as well.

Reply:
Will address this